Task_02/ex2.py

```diff
@@ -0,0 +1,23 @@
+"""Price."""
+
+
+def main():
+    """Main."""
+    instr = open('input.txt', 'r').read()
+    [dollar, cent, count] = get_nums(instr)
+    print(calculate_price(dollar, cent, count))
+
+
+def get_nums(instr):
+    """Get numbers from string."""
+    return [int(s) for s in instr.split() if s.isdigit()]
+
+
+def calculate_price(dollar, cent, count):
+    """Price calculation."""
+    return 'Total cost: {0} dollars {1} ' \
```

Comment:
It's just a tip, you shouldn't fix this:
If string looks too huge you can easily do smth like this: 
`MESSAGE_TMPL = 'total cost ... {dollars}... {cents}...'`
`return MESSAGE_TMPL.format(dollars=..., cents=...)`
If it still looks huge it can be 
`message = ...`
`dollars = ...`
`cents = ...`
`return message.format(dollars=dollars, cents=cents)`

Reply:
Thanks. Need I fix it?

Reply:
No, it's just a tip for your future code.